docs_src/src/pages/documentation/example_app/index.mdx

```diff
@@ -65,3 +78,87 @@ And he was done! The Robyn CLI created a new application with the following stru
   />
 </div>
 
+If you choose to go with the structured scaffold, this is how your project will look like
+
+> note: at the moment, only no-db and sqlalchemy are supported here, you can always plug in other integrations as you see fit
+
+```bash
+├── adaptors
+│   ├── __init__.py
+│   ├── models
+│   │   ├── __init__.py
+│   │   └── user.py
+│   ├── mutators
+│   │   └── __init__.py
+│   └── selectors
+│       ├── __init__.py
+│       └── misc.py
+├── alembic.ini
+├── api
+│   ├── handlers
+│   │   ├── __init__.py
+│   │   ├── probes.py
+│   │   └── sample.py
+│   └── middlewares
+│       └── __init__.py
+├── conf.py
+├── config.env
+├── devops
+│   ├── Dockerfile
+│   ├── Dockerfile.src
+│   └── docker-compose.yaml
+├── migrations
+│   ├── README
+│   ├── env.py
+│   ├── script.py.mako
+│   └── versions
+│       └── __init__.py
+├── requirements.txt
+├── server.py
+└── utils
+    ├── __init__.py
+    └── db.py
+
+12 directories, 24 files
+```
+
+Here's what each of these stand for
+
+- server.py
+
+This is where you instantiate your robyn server and inject global dependencies
+
+```python
+from robyn.helpers import discover_routes
+from robyn import Robyn
+
+from utils.db import get_pool
+from conf import settings
+
+app: Robyn = discover_routes("api.handlers")
```

Comment:
we don't have a discover_routes function

Reply:
It's added in helpers as part of this PR...

```python
def discover_routes(handler_path: str = "api.handlers") -> Robyn:
    mux: Robyn = Robyn(__file__)
    package = importlib.import_module(handler_path)
    for _, module_name, _ in pkgutil.iter_modules(package.__path__, package.__name__ + "."):
        module = importlib.import_module(module_name)
        logger.info(f"member: {module}")
        mux.include_router(module.router)
    return mux
```